### PR TITLE
BaSP-MR-95: tech debt mr 64

### PR DESCRIPTION
**Pull request by ghost**
this PR is an overdue technical debt on ticket MR-64

### CODE REVIEW

**Comment by lorenzomauro59** [comment]
[Tech Debt of MR-64: Subscription Add/Edit Form](https://trello.com/c/sfm5m0PC/95-tech-debt-of-mr-64-subscription-add-edit-form)

File: src/Components/Subscriptions/Form/index.jsx [hunk 1]
```diff
@@ -32,6 +34,10 @@ const Form = ({
     }));
   };
 
+  const handleCancel = () => {
+    setShowForm(false);
+  };
+
   const onSubmit = (e) => {
     e.preventDefault();
```
Comment (fernandofernandezuy) [inline src/Components/Subscriptions/Form/index.jsx]: You leave two lines empty the 43 and 46

File: src/Components/Subscriptions/Form/form.module.css [hunk 2]
```diff
@@ -1,33 +1,57 @@
-form {
+.form-container{
```
Comment (fernandofernandezuy) [inline src/Components/Subscriptions/Form/form.module.css]:
```suggestion
.form-container {
```

File: src/Components/Subscriptions/Form/index.jsx (hunk 1 above)
Comment (ghost) [reply src/Components/Subscriptions/Form/index.jsx]: Thanks Fer. I'll fix it tonight

**Review by fernandofernandezuy — APPROVED** [review]
Seems good

**Review by lorenzomauro59 — APPROVED** [review]
It's approved!

**Comment by araefedg** [comment]
Found inconsistencies in the error modal! leave you a ticket on trello!  https://trello.com/c/h07cTV9V

**Pull request merged**